src/server/server.ts

```diff
@@ -317,6 +318,7 @@ function compileCommon(req: express.Request, res: express.Response, wrapWithRunn
             }
 
             timer.stop();
+            res.setHeader('Cache-Control', 'no-cache, no-store');
```

Comment:
POST responses are not cacheable unless cache-control headers are set, so this may not be necessary.